src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const periodDiscounts = { 3: 20, 7: 50 };
+
+  const discount = getDiscount(days, periodDiscounts);
+
+  return days * pricePerDay - discount;
```

Comment:
The function `calculateRentalCost` is well-named and its purpose is clear. However, the use of an object `periodDiscounts` with hardcoded values is not in line with the checklist item 2. It's better to define these as constants outside of the function scope.